test/tracer_logger.spec.js

```diff
@@ -25,23 +24,23 @@ describe('logger', () => {
   it('should set logger as rewriter logger with ERROR level', () => {
     const rewriter = new Rewriter({ logLevel: 'ERROR', logger })
 
-    rewriter.rewrite('function concat(a, b){ return a + b }', 'test.js')
+    rewriter.rewrite('function concat(a, b){ return a + b }', 'test.js', ['iast'])
 
     expect(logger.error).to.have.been.calledOnceWith('Wasm js rewriter logger configured OK')
   })
 
   it('should set logger as rewriter logger with DEBUG level', () => {
     const rewriter = new Rewriter({ localVarPrefix: 'logger-test', logLevel: 'DEBUG', logger })
 
-    rewriter.rewrite('function concat(a, b){ return a + b }', 'test.js')
+    rewriter.rewrite('function concat(a, b){ return a + b }', 'test.js', ['iast'])
 
     expect(logger.debug).to.have.been.called
     expect(logger.debug.firstCall.args).to.deep.eq(['Wasm js rewriter logger configured OK'])
     expect(logger.debug.secondCall.args).to.deep.eq([
       'Rewriting js file: test.js with config: \
 Config { chain_source_map: false, print_comments: false, local_var_prefix: "logger-test", csi_methods: \
 CsiMethods { methods: [], plus_operator: None, tpl_operator: None, method_with_literal_callers: [] }, \
-verbosity: Information, literals: true }'
+verbosity: Information, literals: true, strict: false }'
```

Comment:
I actually missed this before. Any reason that had to be changed?

Reply:
Hmm @dubloom this is your change. If it's no longer needed, can you rever it?

Reply:
Strict is a new configuration. It allows to chose the behavior when a pass is failing:
- on strict mode, a pass fails, we stop the rewriting
- on non strict mode, a pass fails, we remove it and we start from scratch.

Because of this line: https://github.com/DataDog/dd-wasm-js-rewriter/blob/6e99d0f01f98124549cea4887a1e075178defa9a/src/rewriter.rs#L108

It is logged so we had to update the tests

Reply:
So these tests need to start from scratch? I don't understand how a second pass would change anything. Since we rewrite things, I would normally expect the outcome to be 100% consistent without any code changes.